locales/en-US/notes.properties

```diff
@@ -17,7 +17,7 @@ disableSync=Disable Sync
 reconnectSync=Reconnect to Sync
 syncError=Sync Error
 
-savingChanges=Saving changes…
+savingChanges=Syncing…
```

Comment:
If you change a string you also need to change the associated ID
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

Also make sure to update the reference in comments (if available)